### PR TITLE
Improve tests of liquidation_handler contract

### PR DESCRIPTION
**Pull request by tevrat-aksoy**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Testing

Resolves: #NA

#479

### CODE REVIEW

File: tests/exchange/test_liquidation_handler.cairo [hunk 1]
```diff
@@ -23,106 +26,433 @@ use satoru::exchange::base_order_handler::BaseOrderHandler::{
     event_emitter::InternalContractMemberStateTrait, data_store::InternalContractMemberStateImpl
 };
 use satoru::event::event_emitter::{IEventEmitterDispatcher};
+use satoru::data::{data_store::{IDataStoreDispatcher, IDataStoreDispatcherTrait}, keys};
+use satoru::oracle::oracle::{Oracle, IOracleDispatcher, IOracleDispatcherTrait};
+use satoru::utils::precision;
+use satoru::price::price::Price;
+use satoru::oracle::oracle_store::{IOracleStoreDispatcher, IOracleStoreDispatcherTrait};
+use satoru::oracle::{interfaces::account::{IAccount, IAccountDispatcher, IAccountDispatcherTrait}};
```
Comment (zarboq) [inline tests/exchange/test_liquidation_handler.cairo:34]: merge imports that have the same path

File: src/exchange/liquidation_handler.cairo [hunk 2]
```diff
@@ -41,14 +41,18 @@ mod LiquidationHandler {
 
     // Core lib imports.
     use satoru::exchange::base_order_handler::BaseOrderHandler::{
```
Comment (zarboq) [inline src/exchange/liquidation_handler.cairo:43]: You can put this under // Local imports

**Review by zarboq — CHANGES_REQUESTED** [review]
Small fixes then good to go

File: src/exchange/base_order_handler.cairo [hunk 3]
```diff
@@ -35,6 +35,8 @@ trait IBaseOrderHandler<TContractState> {
         swap_handler_address: ContractAddress,
         referral_storage_address: ContractAddress
     );
+
+    fn only_liquidation_keeper(self: @TContractState);
```
[the remark this answers is not in the document]
Comment (tevrat-aksoy) [reply src/exchange/base_order_handler.cairo]: fixed here
https://github.com/tevrat-aksoy/satoru/commit/c5354902dd17997a0ff6b24971a6204b4e1c07c0

**Review by zarboq — APPROVED** [review]

**Comment by zarboq** [comment]
LGTM :rocket:

**Pull request merged**